dal/dal.py

```diff
@@ -0,0 +1,94 @@
+from mongodb.mongodb_handler import (
+    update_item_status,
+    update_patron_items,
+    insert_document,
+    delete_document, delete_bill, update_bill, insert_bill
+)
+
+
+def update_item_dal(isbn, is_borrowed):
+    """
+    Update the status of a library item identified by its ISBN.
+
+    Args:
+        isbn (str): The ISBN of the library item.
+        is_borrowed (bool): True if the item is borrowed, False if it is returned.
+
+    """
+    update_item_status(isbn, is_borrowed)
```

Comment:
If the only purpose of this function is to call another function. It has no reason to exist (in this case).
It is misleading to call this function `update_item` when all it does is change status.
If I wanted to rename a book or change an author, Intuitively, I would call this function